src/csharp/Microsoft.Spark/Utils/AssemblyLoader.cs

```diff
@@ -189,12 +189,12 @@ private static bool TryLoadAssembly(string assemblyFileName, ref Assembly assemb
         {
             foreach (string searchPath in s_searchPaths.Value)
             {
-                string assemblyPath = Path.Combine(searchPath, assemblyFileName);
-                if (File.Exists(assemblyPath))
+                FileInfo assemblyFile = new FileInfo(Path.Combine(searchPath, assemblyFileName));
+                if (assemblyFile.Exists)
                 {
                     try
                     {
-                        assembly = LoadFromFile(assemblyPath);
+                        assembly = LoadFromFile(assemblyFile.FullName);
```

Comment:
Instead of making modifications here, we can instead change line 63.
from:
```csharp
searchPaths.Add(sparkFilesPath);
```
to:
```csharp
searchPaths.Add(Path.GetFullPath(sparkFilesPath));
```

Reply:
I think this is a better place since we have multiple calls to `searchPaths.Add`.